web/styles/search.css

```diff
@@ -52,6 +52,10 @@
             opacity: 1;
         }
 
+        &:focus-visible {
+            outline-offset: -3px;
+        }
```

Comment:
Hmm, just noticed after merging: Could we also clean up the `&:not(:focus-visible)` rule a few lines above to just be an `outline: none` on the main class with an override here?

Reply:
Is this what you mean? #31675